packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -43,7 +43,11 @@ export const REMOVE_WISHLIST_ITEM_CONFIRMATION_DIALOG_CONFIG = {
  * Renders secondary actions on a product-item card in the form of a button group.
  * Represents other actions you want the user to perform with the product-item (eg.: Remove or Edit)
  */
-const WishlistSecondaryButtonGroup = ({productListItemId, onClick = noop}) => {
+const WishlistSecondaryButtonGroup = ({
+    productListItemId,
+    onActionStart = noop,
+    onActionEnd = noop
```

Comment:
if `onClick` is a promise, then the consumer will know when the action ends. An example in our code base is the [action card component](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/components/action-card/index.jsx#L19)